src/shuffle.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
```

Comment:
What do you think you renaming this file to `utils`, or put it inside an `utils` folder?

Some other utils functions might appear and it would be good to have them all together

Reply:
:+1: I just pulled out these utilities w/o much thought to where they should go, will rearrange utils into a `utils` folder